docs/private_shard/architecture.md

```diff
@@ -12,31 +13,35 @@ Due to the fact that by default the network’s token doesn’t have any monetar
-
 For example, if a private shard is run by 100 validators owned by 100 different companies, it can be configured so that every single one of them has 1 percent of the total stake distributed. In this case every company will produce 1% of the blocks and more than 66% of the companies will need to behave maliciously and collude to corrupt the shard and 33% + 1 to stale the network.
-
 In the future we plan to implement different validator staking mechanisms. Being an owner of an NFT could be an entry ticket to become an validator in the shard. The NFT can represent anything from ownership of a game to having gone through KYC.
 
 ## Consensus
 Calimero is using the NEAR consensus algorithm called Nightshade. It is modeled as a single blockchain, in which each block logically contains all the transactions for all the shards, and changes the whole state of all the shards. Physically, however, no participant downloads the full state or the full logical block. Instead, each participant of the network only maintains the state that corresponds to the shards that they validate transactions for, and the list of all the transactions in the block is split into physical chunks, one chunk per shard. You can find more information about how it works on: https://near.org/papers/nightshade/#nightshade.
 
 ## Proof of Authority Shard
+In a proof of authority (PoA) shard, the validation of transactions and blocks is performed by a set of pre-approved validators or nodes, rather than through a competitive mining process. Validators are identified by their reputation, identity, or stake in the network, and are responsible for verifying the authenticity and validity of transactions. PoA shards are often used in private or permissioned blockchain networks, where the validators are known and trusted entities, such as consortium members or enterprise partners. PoA shards offer faster transaction speeds and lower energy consumption compared to proof of work (PoW) or proof of stake (PoS) consensus mechanisms, but may be more centralized and vulnerable to collusion or attacks by the validators.
 
 ## Proof of Stake Shard
+In a proof of stake (PoS) shard, validators are chosen based on their stake in the network, rather than their computational power or identity. Validators are required to lock up a certain amount of cryptocurrency or token as collateral, which incentivizes them to act honestly and secure the network. In a PoS shard, validators take turns proposing and validating blocks, and are rewarded with transaction fees or newly minted tokens. PoS shards are designed to be more energy-efficient, secure, and decentralized than PoW shards, as they do not require large amounts of computational power and electricity to perform the validation process. PoS shards are often used in public blockchain networks, such as Ethereum 2.0.
```

Comment:
Calimero doesn't have a token itself, so PoS in our case just provides users to have different amount of voting power/blocks produced in the network. Essentially what I would mention in the PoA network every validator gets to produce the same amount of blocks and have same voting power, while in the PoS network this can be customised based on the preferences of the network owners.

Reply:
Also, Calimero does not have fees and tokens so this part doesn't apply to "In a PoS shard, validators take turns proposing and validating blocks, and are rewarded with transaction fees or newly minted tokens."